stdlib/3/typing.pyi

```diff
@@ -338,28 +338,28 @@ class MappingView:
     def __len__(self) -> int: ...
 
 class ItemsView(MappingView, AbstractSet[Tuple[_KT_co, _VT_co]], Generic[_KT_co, _VT_co]):
-    def __and__(self, o: Iterable[_T]) -> AbstractSet[Union[Tuple[_KT_co, _VT_co], _T]]: ...
-    def __rand__(self, o: Iterable[_T]) -> AbstractSet[Union[Tuple[_KT_co, _VT_co], _T]]: ...
+    def __and__(self, o: Iterable[_T]) -> Set[Tuple[_KT_co, _VT_co]]: ...
+    def __rand__(self, o: Iterable[_T]) -> Set[_T]: ...
```

Comment:
Should be `Set[Intersection[_T, Tuple[_KT_co, _VT_co]]]`, but since there is not intersection type currently (cf. python/typing#213), your version is the best we can do, I think.